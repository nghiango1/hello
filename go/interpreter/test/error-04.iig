// Unfinish function
let x = fn (x, 
