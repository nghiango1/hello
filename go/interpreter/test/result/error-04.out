Errors when parsing:
	Expected next token to be ), got EOF instead
